src/hooks/FilterData.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+
+const FilterData = () => {
+  const { count, endDate } = useSelector((store) => store.fakeapidata);
+  const data = useSelector((store) => store.fakeapidata.data);
+  const startDate = useSelector((store) => store.fakeapidata.startDate);
+
+  const [filteredData, setFilteredData] = useState(data);
+
+  const filterData = () => {
+    let newFilteredData = data;
+
+    if (count) {
+      newFilteredData = data?.filter((item) => {
+        if (count === "below 100" && item?.rating?.count <= 100) return item;
```

Comment:
@9xVibee can you change this name from count to more relevant to the filter plus can you move this string check to constants?